core/esmf-aspect-meta-model-java/src/main/java/org/eclipse/esmf/aspectmodel/loader/AspectModelLoader.java

```diff
@@ -173,6 +178,37 @@ public AspectModel load( final InputStream inputStream ) {
       return buildAspectModel( loaderContext.loadedFiles() );
    }
 
+   public AspectModel loadFromArchive( final String pathToArchive ) {
```

Comment:
IMO we should use the terminology from [ADR-0009](https://github.com/bci-oss/esmf-semantic-aspect-meta-model/blob/305-specify-namespace-packages/documentation/decisions/0009-namespace-packages.md) and not talk about "archive" but "namespace package". Also, I'd align this with existing API and provide loading methods for `File`, `InputStream` and `AspectModelUrn` (but not `String`).

```suggestion
   public AspectModel loadNamespacePackage( final File namespacePackage ) {
```